### PR TITLE
CNV-39662: Document HCP KubeVirt GPU device usage

### PR DESCRIPTION
**Pull request by davidvossel**
This documents how to add GPU devices to HCP KubeVirt NodePools

### CODE REVIEW

File: docs/content/how-to/kubevirt/gpu-devices.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Note that the `--host-device-name` argument can be specified multiple times
+in order to attach multiple devices of different types.
+
+## Attaching GPU Devices via the NodePool API Directly
+
+The same example provided above with the hcp cli can also be expressed
+directly using the NodePool API. Below is an example of what the yaml
+would look like to configure a NodePool with 3 replicas with 2 GPU devices
+for each replica. Take note of the `np.spec.platform.kubevirt.hostDevices`
```
Comment (jparrill) [inline docs/content/how-to/kubevirt/gpu-devices.md:54]: nit: s/np.spec.platform.kubevirt.hostDevices/nodepool.spec.platform.kubevirt.hostDevices/

**Review by jparrill — COMMENTED** [review]
small nit, otherwise lgtm

File: docs/content/how-to/kubevirt/gpu-devices.md (hunk 1 above)
Comment (davidvossel) [reply docs/content/how-to/kubevirt/gpu-devices.md]: fixed, thanks

**Pull request merged**